pagopa/pn-frontend
# Pn 5872 fe distinguere gli aar in caso di destinatari multipli mittente

## Short description
Added recipients data in PA next to AAR documents name.

## List of changes proposed in this pull request
- Added recipient denomination and tax id next to AAR document name

## How to test
Open a multi recipient notification in PA and look for Other documents box. You should see AAR title documents with recipient name and tax id.
